codecarbon/emissions_tracker.py

```diff
@@ -293,9 +323,13 @@ def __init__(
                     f"Exception occured {e}"
                 )
 
-        self.country_2letter_iso_code: Optional[str] = (
-            country_2letter_iso_code.upper() if country_2letter_iso_code else None
-        )
+        self.country_2letter_iso_code: Optional[
+            str
+        ] = country_2letter_iso_code or config.get("country_2letter_iso_code")
```

Comment:
```suggestion
        self.country_2letter_iso_code: Optional[str] = (
            country_2letter_iso_code or config.get("country_2letter_iso_code")
        )
```

Reply:
unfortunately this is not on me but on `black`

Reply:
They were not kidding when they said that
> Black is a PEP 8 compliant opinionated formatter. 

You must bit quite opinionated to split time hints this way.